API/schema/message-schema.txt

```diff
@@ -1,31 +1,54 @@
+//Database
 //Our message system will limit to 2 interlocutors
 {
     //The ultimate storage of all the conversations between users
     chatStorage: [
         //One conversation between 2 people, including all their chat history and messages
         {
             //This array includes the names of the two interlocutors
-            interlocutors: ["name1","name2"], 
+            interlocutors: [str, str], (alphabetical)
 
             //This array stores all the messages between the two interlocutors
             chatHistory: [
-                {"chatTime": "18:30 2020/08/19", "chatSender": "name1", "chatContent": "HI Bill!"},
-                {"chatTime": "18:34 2020/08/19", "chatSender": "name2", "chatContent": "HI, Kerwin!"},
-                {"chatTime": "18:35 2020/08/19", "chatSender": "name2", "chatContent": "What's up?"}
+                {"chatTime": datetime, "chatSender": str (interlocuters[i]), "chatContent": str},
+                ...
             ]
         }
     ],
+}
 
+//Example
+{
+    chatStorage: [
+        {
+            interlocutors: ["Bill Gu", "Kerwin Lin"], 
+            chatHistory: [
+                {"chatTime": "18:30 2020/08/19", "chatSender": "Kerwin Lin", "chatContent": "HI Bill!"},
+                {"chatTime": "18:34 2020/08/19", "chatSender": "Bill Gu", "chatContent": "HI, Kerwin!"},
+                {"chatTime": "18:35 2020/08/19", "chatSender": "Bill Gu", "chatContent": "What's up?"}
+            ]
+        }
+    ],
+}
 
+//"/messages" API endpoint
+//Params
+sender1=str
+sender2=str
+numResults=int
```

Comment:
Only want to return the last `n` messages at a time, and if they want to load more, we just make the param larger